src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] arrayOfFigure = new Figure[6];
+        final int First_Half_Of_Array = 3;
```

Comment:
Constant names should be in uppercase snake case, for example, 'FIRST_HALF_OF_ARRAY'. Moreover, all magic numbers in your code should be constants. Please define a constant for the number '6' used in the array initialization as well.